sktime/forecasting/base/tests/test_fh.py

```diff
@@ -335,7 +343,7 @@ def test_relative_to_relative_with_integer_horizon(freqstr):
     """Test converting between relative and absolute with integer horizons."""
     # Converts from relative to absolute and back to relative
     train = pd.Series(1, index=pd.date_range("2021-10-06", freq=freqstr, periods=3))
-    fh = ForecastingHorizon([1, 2, 3])
+    fh = ForecastingHorizon([1, 2, 3], freq=freqstr)
```

Comment:
does this not work without `freq`?
Should it not pick up the frequency now from `train.index`?

Reply:
Unfortunately, no. `train.index` is `pd.DatetimeIndex`, `train.index[-1]` is `pd.Timestamp` which is precisely what we don't want to be the source of frequency information.

Reply:
yes, you are right - that would not work.
Still, not passing `freq` *should* work for the user. This test, in #2932, has the old syntax for `fh` construction.